react/features/chat/actions.web.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+export * from './actions.any';
+
+/**
+ * Displays a display name input prompt.
+ *
+ * @returns {openDialog}
+ */
+export function displayChatNameInputDialog() {
+    // Not used on web, just here for flow compatibility.
```

Comment:
Web does have a display name prompt, if you want to slide that in here.

Reply:
I saw it, but since that's a form component, I wasn't sure how to abstract this different behaviour. Or you mean I should use a dialog on web too to share behaviour?

Reply:
There's a separate display name prompt. It doesn't look like a form to me. I meant you could have a separate display name prompt, probably independent of chat. There is some server configuration that requires a display name so that web display name prompt exists. As such, instead of leaving an empty function, could change the function to actually display the existing prompt.

Reply:
Did that. It was painful. But thanks for reminding me for these!